Calling `get_tags_in_window` from Python is broken in 3.9

It doesn't add the offset for the number of previously read samples, so in effect it acts exactly like `get_tags_in_range`. This only happens when the function is called on a Python block, not with any of the C blocks.

I can produce a failing test with this behavior by modifying the [`qa_block_gateway.py`](https://github.com/gnuradio/gnuradio/blob/1f1733eb4489b48fb73509e7806df19e1c738092/gr-blocks/python/blocks/qa_block_gateway.py#L125) script so that the `tag_source` skips outputting a tag in the first work call:
```
        # put code here to fill the output items...
        if self.nitems_written(0) == 0:
            # skip tagging in the first work block
            return num_output_items
```
Resulting in:
```
$ python qa_block_gateway.py 
........F
======================================================================
FAIL: test_tags_win (__main__.test_block_gateway)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/rvolz/repos/gnuradio/gr-blocks/python/blocks/qa_block_gateway.py", line 287, in test_tags_win
    self.assertEqual(sink.key, "example_key")
AssertionError: None != 'example_key'

----------------------------------------------------------------------
Ran 9 tests in 0.355s

FAILED (failures=1)
```

I haven't tracked down a cause for this, but presumably it happened with the SWIG->pybind11 transition since I see it with 3.9.2.0 but not 3.8.3.1. The easy workaround is to just use `get_tags_in_range` instead and add the offset manually.

--- a/gnuradio-runtime/python/gnuradio/gr/bindings/block_gateway_python.cc
+++ b/gnuradio-runtime/python/gnuradio/gr/bindings/block_gateway_python.cc
@@ -67,7 +67,7 @@ void bind_block_gateway(py::module& m)
         .def(
             "get_tags_in_window",
             (std::vector<gr::tag_t>(block_gateway::*)(unsigned int, uint64_t, uint64_t)) &
-                block_gateway::_get_tags_in_range,
+                block_gateway::_get_tags_in_window,
             py::arg("which_input"),
             py::arg("rel_start"),
             py::arg("rel_end"))
@@ -75,7 +75,7 @@ void bind_block_gateway(py::module& m)
         .def("get_tags_in_window",
              (std::vector<gr::tag_t>(block_gateway::*)(
                  unsigned int, uint64_t, uint64_t, const pmt::pmt_t&)) &
-                 block_gateway::_get_tags_in_range,
+                 block_gateway::_get_tags_in_window,
              py::arg("which_input"),
              py::arg("rel_start"),
              py::arg("rel_end"),

--- a/gr-blocks/python/blocks/qa_block_gateway.py
+++ b/gr-blocks/python/blocks/qa_block_gateway.py
@@ -122,7 +122,10 @@ class tag_source(gr.sync_block):
         num_output_items = len(output_items[0])
 
         # put code here to fill the output items...
-
+        if self.nitems_written(0) == 0:
+            # skip tagging in the first work block
+            return num_output_items
+            
         # make a new tag on the middle element every time work is called
         count = self.nitems_written(0) + num_output_items // 2
         key = pmt.string_to_symbol("example_key")

